packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -132,10 +132,17 @@ const Label = createComponentSlot(FormLabel)<FieldContainerClassKey, OwnerState>
             margin-top: ${theme.spacing(2)};
         `}
 
-        ${ownerState.disabled &&
-        css`
-            color: ${theme.palette.text.disabled};
-        `}
+        &.Mui-disabled {
+            color: ${theme.palette.text.primary};
+        }
+
+        &.Mui-focused {
+            color: ${theme.palette.text.primary};
+        }
+
+        > .MuiFormLabel-asterisk {
+            margin-left: 4px;
+        }
```

Comment:
I think these styles should apply to all labels, not just the ones in `FieldContainer`. 
Could you do this in the theme styling of MUIs `FormLabel`?

https://github.com/vivid-planet/comet/blob/main/packages/admin/admin-theme/src/componentsTheme/MuiFormLabel.ts